htdocs/libraries/icms/core/HTMLFilter.php

```diff
@@ -50,29 +50,29 @@ static public function filterHTML($html) {
 		$icmsConfigPurifier = icms::$config->getConfigsByCat(\icms_config_Handler::CATEGORY_PURIFIER);
 
         $fcomment = '<!-- filtered with htmlpurifier -->';
-        
+
         $purified = strpos($html, $fcomment);
         if ($purified !== FALSE) {
             $html = str_replace($fcomment, '', $html);
         }
 
         if ($icmsConfigPurifier['enable_purifier'] !== 0) {
-			require_once ICMS_LIBRARIES_PATH . '/htmlpurifier/HTMLPurifier.standalone.php';
-			require_once ICMS_LIBRARIES_PATH . '/htmlpurifier/HTMLPurifier.autoload.php';
+//			require_once ICMS_LIBRARIES_PATH . '/htmlpurifier/HTMLPurifier.standalone.php';
```

Comment:
I think it would be better to remove these lines - not to comment. Because it's not very usefull.